src/hints/bls_utils.rs

```diff
@@ -0,0 +1,50 @@
+use cairo_vm::vm::errors::hint_errors::HintError;
+use cairo_vm::Felt252;
+use lazy_static::lazy_static;
+use num_bigint::BigInt;
+use num_integer::Integer;
+use num_traits::Signed;
+
+lazy_static! {
+    static ref BASE: BigInt = BigInt::from(2).pow(86);
+}
+
+/// Takes an integer and returns its canonical representation as:
+///    d0 + d1 * BASE + d2 * BASE**2.
+/// d2 can be in the range (-2**127, 2**127).
+pub fn split(num: Felt252) -> Result<Vec<Felt252>, HintError> {
```

Comment:
What kind of canonical representation is this?

Reply:
That comment was copied from `cairo-lang`. I'll take suggestions on improving it... :shrug: